src/services/completions.ts

```diff
@@ -766,7 +770,9 @@ namespace ts.Completions {
             // For JavaScript or TypeScript, if we're not after a dot, then just try to get the
             // global symbols in scope.  These results should be valid for either language as
             // the set of symbols that can be referenced from this location.
-            if (!tryGetGlobalSymbols()) {
+
+            const meanings = SymbolFlags.Type | SymbolFlags.Namespace | SymbolFlags.Alias;
+            if (!tryGetGlobalSymbols(isTypeLocation ? meanings : undefined)) {

```

Comment:
Dont think this is the place to filter meaning.. You want to do that in filterGlobalCompletion instead

Reply:
fixed